src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -7,7 +7,13 @@ namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
+#if PLATFORM_NETCORE50AOT
+        private const string FrameworkName = ".NET Native";
+#elif PLATFORM_NET46
+        private const string FrameworkName = ".NET Full Framework";
```

Comment:
Can be called anything, added full to imply the full desktop. can change to just .net framework.